ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/GoIntoAction.java

```diff
@@ -59,6 +59,6 @@ public void actionPerformed(ActionEvent e) {
     }
 
     private boolean isNodeSupportGoInto(Object node) {
-        return node != null && node instanceof Node && ((Node)node).getParent() != null && ((Node)node).supportGoInto();
+        return node != null && node instanceof Node && ((Node)node).supportGoInto();
```

Comment:
checking for nullable parent was added specially, cause going into project node simply has no meaning